lib/bitcoin/BitcoinRawDataParser.ts

```diff
@@ -54,7 +60,30 @@ export default class BitcoinRawDataParser {
         throw new SidetreeError(ErrorCode.BitcoinRawDataParserInvalidBlockData);
       }
 
-      blockMapper[block.hash] = block;
+      // the first transaction, the coinbase, contains the block height in its input
+      const coinbaseInputScript = (block.transactions[0].inputs[0] as any)._scriptBuffer;
+      // this denotes how many bytes following represent the block height
+      const heightBytes = coinbaseInputScript.readUInt8();
+      // the next n bytes are the block height in little endian
+      const blockHeight = coinbaseInputScript.readUIntLE(1, heightBytes);
+
+      const transactionModels = block.transactions.map((transaction: any) => {
+        const bitcoreTransaction = {
+          id: transaction.id,
+          blockHash: block.hash,
+          confirmations: 1, // set to 1 because it has to be confirmed to be on the current longest chain
```

Comment:
Looks like you are just assigning a value here because you have to, if so consider explicitly say so in the comments.
```suggestion
          confirmations: 1, // Unused, but need to set to a value to be able reuse `BitcoinTransactionModel`.
```
Alternatively, you could have `BitcoinTransactionModel` and `ConfirmedBitcoinTransactionModel`, but I would agree that's probably for annoying than helpful.